packages/Calendar/src/Calendar/Day.js

```diff
@@ -1,26 +1,21 @@
 // @flow
 import React from 'react';
 import { View, TouchableOpacity, Text, StyleSheet } from 'react-native';
-import { getTheme } from '@sharingapples/theme';
+import { textColor, backgroundColor, primaryFontColor, primaryColor, disabledFontColor } from '../theme';
 
-const theme = getTheme();
-const calendarTheme = theme.onCalendar || theme;
-const textColor = calendarTheme.onBackground;
-const backgroundColor = calendarTheme.background;
-const primaryFontColor = theme.onPrimary;
-const disabledFontColor = theme.disabled;
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
     alignItems: 'center',
     paddingHorizontal: 3,
     paddingVertical: 3,
-    borderLeftWidth: 1,
-    borderRightWidth: 1,
-    borderTopWidth: 1,
-    borderBottomWidth: 1,
-    margin: -1,
+    borderWidth: StyleSheet.hairlineWidth,
+    margin: -StyleSheet.hairlineWidth,
+    borderLeftColor: 'transparent',
```

Comment:
May be you would want to put a comment here. As far as I reckon, this has something to do with android inconsistency

Reply:
huss